packages/ove-app-charts/README.md

```diff
@@ -0,0 +1,41 @@
+# Charts App
+
+This app supports visualisation of charts (excluding visualisation of networks) using the OVE framework. It is based on [Vega-Embed](https://github.com/vega/vega-embed), which provides support for both [Vega](https://vega.github.io/vega/) and [Vega-Lite](https://vega.github.io/vega-lite/): these are both declarative languages for interactive visualizations, but Vega-Lite is a higher level language. Vega-Embed supports Canvas and SVG rendering.
```

Comment:
Technically it does support the drawing of networks - see [this example](https://vega.github.io/vega/examples/force-directed-layout/).

I suggest deleting "(excluding visualisation of networks)" and adding a sentence saying that OVE also provides a separate Networks App that is specialised for drawing node-link diagrams.

Reply:
This was included because OVE's charts app was not intended to be used to support networks. I will add a separate line.